contrib/inventory/scaleway.ini

```diff
@@ -0,0 +1,14 @@
+[scaleway]
+api_token = 
```

Comment:
A brief introduction to each parameter would be very helpful to new person. I would expect at least one liner for each parameter.